showcase-docs/getting-started.md

```diff
@@ -196,6 +226,13 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
      - `${AZURE_TOKEN}`: personal access token
      - `${AZURE_ORG}`: Azure DevOps Services (cloud) Organization name or the Azure DevOps Server
 
+   - Setup the Jira plugin
+
+     - This [URL](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-jira#how-to-use-jira-plugin-in-backstage) is explains how to use the Jira plugin
+     - `${JIRA_URL}`: URL for the Jira instance
+     - `${JIRA_TOKEN}`: API token
+     - `${AZURE_USER_AGENT}`: User-Agent (UA) string
```

Comment:
It should be `JIRA_USER_AGENT`
Also is this value required?

Reply:
TBH, I am not completely sure its required, the documentation isn't great.